tests/unit/test_utils.py

```diff
@@ -380,6 +404,58 @@ def test_unpack_zip(self, data):
         unzip_file(test_file, self.tempdir)
         self.confirm_files()
 
+    def test_unpack_zip_failure(self):
+        """
+        Test unpacking a *.zip with file containing .. path
+        and expect exception
+        """
+        test_zip = self.make_zip_file('test_zip.zip',
+                                      ['regular_file.txt',
```

Comment:
I would put the files in a separate variable so the call itself can be 1 line. Likewise for the other checks.

Also see [here](https://github.com/pypa/pip/blob/acc895c57ef6c6c7b1cf06f252b267438bd86c89/tests/unit/test_utils.py#L918-L924) for function call style and [here](https://github.com/pypa/pip/blob/acc895c57ef6c6c7b1cf06f252b267438bd86c89/tests/unit/test_utils.py#L1194) for general list declaration style to use if it goes over 1 line.